test/functional/Java8andUp/src/org/openj9/test/attachAPI/TestAttachAPI.java

```diff
@@ -222,13 +216,17 @@ public void test_agntld03() {
 		String[] libDirs = lipPath.split(File.pathSeparator);
 		char fs = File.separatorChar;
 		String decoration = "lib";
-		String suffix = ".so";
-		String errOutput = "";
+		String suffix;
```

Comment:
Default to `= ".so";` and the variable will always have a value and branching below will be simpler